### DISCUSSION TITLE
How to ignore empty lines in sentence segmentation?

### DISCUSSION THREAD

**Discussion by YitongChenDavid** [explosion/spaCy#10408] [Help: Coding & Implementations]
When a txt file has multiple empty lines together or has some 'subtitles', how do we customize the sentence segmentation?

**Comment by polm** [comment]
> how do we customize the sentence segmentation?

You need to make training data and train a model. Usually we recommend making training data by putting once sentence per line in a file and processing that into Docs, but since you seem to want sentences to include newlines you might need to use a different separator character. Alternately, you could replace newlines with spaces before processing your text (I would recommend that approach). 

> How to ignore empty lines in sentence segmentation?

> When a txt file has multiple empty lines together or has some 'subtitles',

I don't understand what exactly you're describing here, could you give some examples?
